### PR TITLE
feat: implement query to get all items in a collection

### PR DESCRIPTION
**Pull request by BryanPan342**
Static functions for all objects to get all the items in their respective collections. This will be useful later when we want to run playlist generation on all the lobbies.

### CODE REVIEW

File: scripts/regenerate-playlists.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { Lobby } from '../lib/lobby';
+
+async function main() {
+  const lobbies = await Lobby.all();
+  lobbies.forEach(lobby => void lobby.synthesizePlaylist());
+}
+
+void main();
```
Comment (trevorong) [inline scripts/regenerate-playlists.ts:8]: is the void main so it runs every time the server restarts but only runs once per restart?

Comment (BryanPan342) [reply scripts/regenerate-playlists.ts:8]: The void is because main is async and returns a promise. Also this only runs whenever we run it!

Comment (trevorong) [reply scripts/regenerate-playlists.ts:8]: Oh yea but isnt the call to main() already executing the function?

Comment (trevorong) [reply scripts/regenerate-playlists.ts:8]: also wait can u add `this.managerName = props.managerName;` to the lobby constructor whoops
```suggestion
void main();
```

Comment (BryanPan342) [reply scripts/regenerate-playlists.ts:8]: Let’s do that in another PR

**Review by trevorong — APPROVED** [review]
LGTM

**Pull request merged**